src/ck-core/ckrdma.C

```diff
@@ -1742,5 +1744,33 @@ void readonlyGetCompleted(NcpyOperationInfo *ncpyOpInfo) {
   // Free ncpyOpInfo allocated inside readonlyGet
   CmiFree(ncpyOpInfo);
 }
+
+void _ncpy_ack_handler(ncpyHandlerMsg *msg) {
```

Comment:
I think that names that start with an underscore are reserved in C/C++. Maybe remove the underscore.

Reply:
I added the underscore because other converse handlers in Charm++ were also named with that format (_initHandler, _processHandler, etc).